profiles/common/modules/features/nexteuropa_dgt_connector/ne_tmgmt_dgt_rules/ne_tmgmt_dgt_rules.module

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * @file
+ * Next Europa TMGMT DGT Rules module file.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function ne_tmgmt_dgt_rules_menu() {
+  $items['admin/config/workflow/ne_tmgmt_dgt_rules'] = array(
```

Comment:
I would put the form in a new tab inside rules (admin/config/workflow/rules/ne_tmgmt_dgt_rules)
What do you think?

Reply:
Done.